pypkgs.txt

```diff
@@ -0,0 +1,6 @@
+pyasn1==0.1.9
```

Comment:
In general, python projects call this file `requirements.txt`. Moreoever  I wouldn't define it in riotdocker but create it directly somewhere in the RIOT codebase.

Reply:
Some of those don't even have anything to do with RIOT directly (e.g. `flake8`) but are just used by CI for checks.

Reply:
Well not really: flake8 could be used locally by a developer to do last time static checks before opening a PR.